http-server/src/main/java/io/strimzi/admin/Constants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.admin;
+
+public class Constants {
+    public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
+    public static final String METADATA_MAX_AGE_CONFIG = "metadata.max.age.ms";
+    public static final String REQUEST_TIMEOUT_MS_CONFIG = "request.timeout.ms";
+    public static final String RETRIES_CONFIG = "retries";
+    public static final String DEFAULT_API_TIMEOUT_MS_CONFIG = "default.api.timeout.ms";
+
+    public static final String SSL_KEYSTORE_PASSWORD_CONFIG = "ssl.keystore.password";
+    public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
+    public static final String SSL_KEYSTORE_TYPE_CONFIG = "ssl.keystore.type";
+
+    public static final String SSL_TRUSTSTORE_PASSWORD_CONFIG = "ssl.truststore.password";
+    public static final String SSL_TRUSTSTORE_LOCATION_CONFIG = "ssl.truststore.location";
+    public static final String SSL_TRUSTSTORE_TYPE_CONFIG = "ssl.truststore.type";
```

Comment:
why are we duplicating constants from Kafka?

Reply:
Well, I was trying to remove kafka dependency. But it seems it is needed because of this single line `new ConfigResource(org.apache.kafka.common.config.ConfigResource.Type.TOPIC, topicToDescribe);`

Reply:
it's kind of strange ... this `Type` is used here in the Vert.x Kafka client so it should be available with the indirect dependency:

https://github.com/vert-x3/vertx-kafka-client/blob/3.9/src/main/java/io/vertx/kafka/client/common/ConfigResource.java